README.md

```diff
@@ -10,16 +14,64 @@
 [![python version](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)](https://img.shields.io/badge/python-3.8%20%7C%203.9%20%7C%203.10%20%7C%203.11-blue)
 [![scikit-learn supported versions](https://img.shields.io/badge/sklearn-1.0%20%7C%201.1%20%7C%201.2%20%7C%201.3-blue)](https://img.shields.io/badge/sklearn-01.0%20%7C%201.1%20%7C%201.2%20%7C%201.3-blue)
 
-With Intel(R) Extension for Scikit-learn you can accelerate your Scikit-learn applications and still have full conformance with all Scikit-Learn APIs and algorithms. This is a **free software AI accelerator** that brings over **10-100X** acceleration across a variety of applications. And you do not even need to change the existing code!
+---
+</h3>
+
+<div align="left">
+
+## Overview
+
+Intel(R) Extension for Scikit-learn is a **free software AI accelerator** designed to deliver up to **100X** faster performance for your existing scikit-learn code.
+The software acceleration is achieved with vector instructions, AI hardware-specific memory optimizations, threading, and optimizations for all upcoming Intel(R) platforms at launch time.
+
+
+With Intel(R) Extension for Scikit-learn, you can:
+
+* Speed up training and inference by up to 100x with the equivalent mathematical accuracy
+* Benefit from performance improvements across different Intel(R) hardware configurations
+* Integrate the extension into your existing Scikit-learn applications without code modifications
+* Continue to use the open-source scikit-learn API
+* Enable and disable the extension with a couple of lines of code or at the command line
+
+Intel(R) Extension for Scikit-learn is also a part of [Intel(R) AI Tools](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html). 
+
+## Installation
+
+To install Intel(R) Extension for Scikit-learn, run:
+
+```
+pip install scikit-learn-intelex
+```
 
-## How it works?
+See all installation instructions in the [Installation Guide](https://intel.github.io/scikit-learn-intelex/latest/installation.html). 
 
-Intel(R) Extension for Scikit-learn offers you a way to accelerate existing scikit-learn code.
-The acceleration is achieved through **patching**: replacing the stock scikit-learn algorithms with their optimized versions provided by the extension.
+## Integration
 
-One of the ways to patch scikit-learn is by modifying the code. First, you import an additional Python package (`sklearnex`) and enable optimizations via `sklearnex.patch_sklearn()`. Then import scikit-learn estimators:
+The software acceleration is achieved through patching. It means, replacing the stock scikit-learn algorithms with their optimized versions provided by the extension.
 
-- **Enable Intel CPU optimizations**
+> [!NOTE]
```

Comment:
i would rather putt this as shorter catchy phrase within main offering or in line above.

Reply:
changed